modules/cluster-logging-deploy-cli.adoc

```diff
@@ -181,11 +181,8 @@ There should be an Elasticsearch Operator in each Namespace. The version number
 
 . Install the Cluster Logging Operator by creating the following objects:
 
-<<<<<<< HEAD
 .. Create an OperatorGroup object YAML file (for example, `clo-og.yaml`) for the Cluster Logging Operator:
-=======
-.. Create an Operator Group object YAML file (for example, `clo-og.yaml`) for the Cluster Logging Operator:
->>>>>>> 3353f1b68 (RHDEVDOCS-2493 Logging: Change name from "cluster logging" to "OpenShift Logging")
+
```

Comment:
@bobfuru The real preview hasn't generated yet, but looking at the file preview it GitHub, it looks like the numbering is off. I think it's because this line needs to be deleted, so that the + keeps the below code block with the above bullet's text.

Reply:
Oh, right. Good catch, thanks!
Resolved